src/types/worker/WorkerDTO.ts

```diff
@@ -1,7 +1,7 @@
 import Worker from './Worker';
 
 type WorkerDTO = Omit<Worker, 'id'> & {
-  password: string
+  password?: string
```

Comment:
should we split into two dtos ? one update that omits password and one create that has it ?